src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -612,14 +639,7 @@ public static void getDependency(PrintStream printStream, String distribution, S
                         Pattern dependencyPattern = Pattern.compile(dependencyRegex);
                         Matcher dependencyMatcher = dependencyPattern.matcher(distInfo);
                         while (dependencyMatcher.find()) {
-                            String dependencyName = dependencyMatcher.group(1);
-                            if (!ToolUtil.checkDependencyAvailable(dependencyName)) {
-                                downloadDependency(printStream, dependencyName, distributionType,
-                                        distributionVersion);
-                            } else {
-                                printStream.println("Dependency '" + dependencyName +
-                                        "' is already available locally");
-                            }
+                            dependencyName = dependencyMatcher.group(1);
```

Comment:
This was written to manage multiple dependencies by looping it and downloading each.

If we are changing like this we should keep dependency names in an array and return it.

Reply:
Here we check for a regexMatcher (`jdk`) within the given distribution details. So we only get a single dependency name always. IINM we don't need an array here.